resources/js/app.js

```diff
@@ -0,0 +1 @@
+import './bootstrap';
```

Comment:
I 100% of the time remove all of the stuff from bootstrap and put it back into this app file.  I think we should do that too.

Reply:
Moved, and README updated